bin/cli.js

```diff
@@ -202,13 +202,39 @@ function invoke(env) {
         .catch(exit);
     });
 
+  commander
+    .command('migrate:up')
+    .description('        Run the next migration.')
```

Comment:
Description seems inaccurate.

Reply:
What are your thoughts on `Run the next chronological migration that has not yet been run.`?